doc/release-notes.md

```diff
@@ -20,3 +20,24 @@ inputs each. Starting from this release, `-mempooltxinputlimit` will be enforced
 before the Overwinter activation height is reached, but will be ignored once
 Overwinter activates. The option will be removed entirely in a future release
 after Overwinter has activated.
+
+`-disabledeprecation` removal
+-----------------------------
+
+In release 1.0.9 we implemented automatic deprecation of `zcashd` software
+versions made by the Zcash Company. The configuration option
+`-disabledeprecation` was added as a way for users to specifically choose to
+stay on a particular software version. However, it incorrectly implied that
+deprecated releases would still be supported.
+
+This release removes the `-disabledeprecation` option, so that `zcashd` software
+versions made by the Zcash Company will always shut down in accordance with the
+defined deprecation policy (currently 16 weeks after release). Users who wish to
+use a different policy must now specifically choose to either:
+
+- edit and compile the source code themselves, or
+- obtain a software version from someone else who has done so (and obtain
+  support from them).
+
+Either way, it is much clearer that the software they are running is not
+supported by the Zcash Company.
```

Comment:
I based these notes on the comments in #2943, but they need review. C/c @zookozcash @ioptio